assets/app/scripts/controllers/builds.js

```diff
@@ -21,6 +22,9 @@ angular.module('openshiftConsole')
     var watches = [];
 
     watches.push(DataService.watch("builds", $scope, function(builds, action, build) {
+      var buildConfigName;
+      var buildName;
```

Comment:
That seems reasonable enough for now, esp since we can punt on this one until the jscs ticket.  

I'm going to leave one more thought on here for us to chew on until then.  I'm a fan of patterns that promote simplicity, and I think this may be one more reason to consider putting all vars at the top of our functions. I like the consistency of knowing a function signature by its variables, and having all functions in our application follow the same pattern helps ensure they are not hiding secrets/bugs. For the variable location: the arguments, and the vars declared at the top of the function.  Its easier to reason about what a function might be doing if I can see its vars without having to search through the function body to find them.  

I'll attach an image, but its prob not needed :smile: 

-----
![function](https://cloud.githubusercontent.com/assets/280512/8456836/ed529e9e-1fdb-11e5-9ffc-9efef74adc5f.png)